tests/test_sentinel.py

```diff
@@ -76,6 +92,16 @@ def sentinel(request, cluster):
     return Sentinel([('foo', 26379), ('bar', 26379)])
 
 
+@pytest.fixture()
+def live_sentinel(is_docker):
+    if is_docker:
```

Comment:
This is another place I use the 'docker' pytest flag -- and here again I guess an alternative is more pytest options?